bears/coffee_script/CoffeeLintBear.py

```diff
@@ -22,11 +22,280 @@ class CoffeeLintBear:
     CAN_DETECT = {'Syntax', 'Formatting', 'Smell'}
 
     severity_map = {'warn': RESULT_SEVERITY.NORMAL,
-                    'error': RESULT_SEVERITY.MAJOR}
+                    'error': RESULT_SEVERITY.MAJOR,
+                    'ignore': RESULT_SEVERITY.INFO}
 
     @staticmethod
     def create_arguments(filename, file, config_file):
-        return '--reporter=raw', filename
+        return '--reporter=raw', '-f', config_file, filename
+
+    @staticmethod
+    def generate_config(filename, file,
+                        max_line_length: int=80,
+                        max_line_length_affect_comments: bool=True,
+                        space_before_and_after_arrow: bool=True,
+                        check_braces_spacing: bool=False,
+                        braces_spacing_width: int=1,
+                        spacing_in_empty_braces: int=0,
+                        class_naming_camelCase: bool=True,
+                        spaces_before_and_after_colon: bool=False,
+                        spaces_before_colon: int=0,
+                        spaces_after_colon: int=0,
+                        enforce_newline_at_EOF: bool=True,
+                        use_spaces: bool=True,
+                        tab_width: int=2,
+                        number_of_newlines_after_classes: int=3,
```

Comment:
though the linter provided a default of 3, I think this is too much too, 2 is good imo :+1: